connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -257,11 +268,12 @@ public Response putConnectorConfig(final @PathParam("connector") String connecto
 
     @POST
     @Path("/{connector}/restart")
+    @Operation(summary = "Restart the specified connector")
```

Comment:
Should we mention `includeTasks` and `onlyFailed` parameters?

Reply:
I've added descriptions for these parameters